content-scripts/src/modules/options/writerMode.js

```diff
@@ -36,7 +36,7 @@ export const changeWriterMode = (writerMode) => {
           `
             body {
               padding-left: 0;
-              overflow: hidden;
+              overflow: scroll;
```

Comment:
@Dineshs91 have you tried completely removing the `overflow` line here? I could work the same

Reply:
Yes, this worked!

I update the PR.